.tests/compatibility/reference/WASH7P.fna

```diff
@@ -0,0 +1,216 @@
+>chr1 WASH7P [organism=Homo sapiens] [GeneID=653635] [chromosome=1]
```

Comment:
why do we need the actual file when nothing is run? To keep the repo small, it should be an empty file.

Reply:
cleaned